tests/Query.test.ts

```diff
@@ -270,6 +270,7 @@ describe('Query parsing', () => {
             'hide start date',
             'hide task count',
             'hide urgency',
+            'hide tags',
```

Comment:
Please add the matching `show tags` command as well. Thanks.
Sorry I missed that earlier.

Reply:
Ah, right, thanks. Added.